app/dashboard/templates/bounty_details.html

```diff
@@ -23,21 +23,7 @@
       </div>
 
       <div class="col-12 offset-md-1 col-md-8 body">
-        <div class="transation-header" style="display:none;">
-          <h3>Your Issue is Posting: </h3>
-          <h3 class="issue-url"></h3>
-        </div>
-        <div class="row loading justify-content-center align-items-center loading_img">
-          <img src="{% static "v2/images/loading_v2.gif" %}" style="max-width: 200px;">
-        </div>
-        <div class="transaction-status" style="display:none;">
-          <div style='width:100%; font-size: 20px; color: #3d3d46;'>
-            {% trans "Your transaction is being posted to the Ethereum blockchain" %}
-            <br>
-            {% trans "Your page will automatically update when the information has been synced to Web3" %}
-          </div>
-        </div>
-
+        {% include 'shared/bounty_hero.html' %}
```

Comment:
could we rename it to something like `bounty_waiting_state.html` or something better ?

Reply:
Okay